src/ctim/schemas/common.clj

```diff
@@ -37,6 +31,54 @@
 
 (def default-tlp "green")
 
+(s/defschema BaseEntity
+  {;; :id and :idref must be implemented exclusively
+   :id ID
+   :type s/Str
+   :schema_version (describe s/Str "CTIM schema version for this entity")
+   (s/optional-key :uri) URI
+   (s/optional-key :revision) s/Int
+   (s/optional-key :external_ids) [s/Str]
+   (s/optional-key :timestamp) Time
+   (s/optional-key :language) s/Str
+   (s/optional-key :tlp) TLP
+   })
+
+(s/defschema NewBaseEntity
+  "Base for New Entities, optionalizes ID and type and schema_version"
+  (st/merge
+   BaseEntity
+   (st/optional-keys
+    {:id ID
+     :type (describe s/Str "A valid entity type identifer")
+     :schema_version (describe s/Str "CTIM schema version for this entity")})))
```

Comment:
Do we want the client to provide a schema version while creating a new entity?

In that case, does that mean that for CTIA we'll have to discard wrong version or keep a list of functions able to transform from one version to another one and make the schema upgrade automatic at creation/update?